sentry_ldap_auth/backend.py

```diff
@@ -19,27 +19,23 @@ def get_or_create_user(self, username, ldap_user):
 
         user.is_managed = True
 
+        # Add the user email address
         try:
             from sentry.models import (UserEmail)
         except ImportError:
             pass
         else:
-            userEmail = UserEmail.objects.filter(user=user)
-            if not userEmail:
-                userEmail = UserEmail.objects.create(user=user)
-            else:
-                userEmail = userEmail[0]
-
-            if not hasattr(settings, 'AUTH_LDAP_DEFAULT_EMAIL_DOMAIN'):
-                email = ' '
-            else:
-                email = username + '@' + settings.AUTH_LDAP_DEFAULT_EMAIL_DOMAIN
-
             if 'mail' in ldap_user.attrs:
-                userEmail.email = ldap_user.attrs.get('mail')[0]
+                email = ldap_user.attrs.get('mail')[0]
+            elif not hasattr(settings, 'AUTH_LDAP_DEFAULT_EMAIL_DOMAIN'):
+                email = ''
```

Comment:
This changes the currently "empty" getsentry-ldap-auth email from `' '` to `''`, possibly leaving dangling database rows with `email=' '`.